src/features/components/molecules/deriv-products-hero/products-logo-and-text.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react'
+import { hero_image } from './deriv-products-hero.module.scss'
+import { DerivProductContentType } from './types'
+import FlexBox from 'features/components/atoms/flex-box'
+
+interface ProductsLogoTextType {
+    contentData: DerivProductContentType
+}
+
+const ProductsLogoAndText: FC<ProductsLogoTextType> = ({
+    contentData: { mobile_logo, product_water_mark_logo_mobile, logo, product_water_mark_logo },
+}) => {
+    return (
+        <FlexBox.Box className={hero_image} justify="start">
+            <FlexBox.Box visible="phone-and-tablet" gap="10x" align="center">
+                {mobile_logo} {product_water_mark_logo_mobile}
```

Comment:
you can use logo together with watemark, just download it from figma

Reply:
sure will do